packages/office-ui-fabric-react/src/components/DatePicker/examples/DatePicker.Basic.Example.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { Dropdown, IDropdownOption } from 'office-ui-fabric-react/lib/Dropdown';
 import { DatePicker, DayOfWeek, IDatePickerStrings } from 'office-ui-fabric-react';
-import './DatePicker.Examples.scss';
+import { mergeStyleSets } from '@uifabric/styling';
```

Comment:
This should be imported from `office-ui-fabric-react/lib/Styling` to match the way we do these imports in other files.